example/src/test/scala/monocle/LensDefExample.scala

```diff
@@ -0,0 +1,41 @@
+package monocle
+
+import monocle.macros.{GenLens, Lenses}
+import monocle.syntax._
+import org.specs2.execute.AnyValueAsResult
+import org.specs2.scalaz.Spec
+import shapeless.test.illTyped
+
+class LensDefExample extends Spec {
```

Comment:
would you mind to merge this example with LensExample.scala, it would be nice to have all the possible ways to create `Lens` in the same place